src/main/java/uk/gov/justice/digital/repository/DomainRepository.java

```diff
@@ -1,26 +1,24 @@
 package uk.gov.justice.digital.repository;
 
 import uk.gov.justice.digital.client.dynamodb.DynamoDBClient;
-import uk.gov.justice.digital.common.Util;
 import uk.gov.justice.digital.domains.model.DomainDefinition;
-import uk.gov.justice.digital.service.DeltaLakeService;
-import java.util.HashSet;
-import java.util.Set;
 
-public class DomainRepository extends Util {
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.PatternSyntaxException;
 
+public class DomainRepository {
 
     private final DynamoDBClient dynamoDBClient;
 
-    protected DeltaLakeService service = new DeltaLakeService();
-
     public DomainRepository(final DynamoDBClient dynamoDBClient) {
         this.dynamoDBClient = dynamoDBClient;
     }
 
 
-    public Set<DomainDefinition> getForName(final String domainTableName, final String domainId) {
+    public Set<DomainDefinition> getForName(final String domainTableName, final String domainId)
+            throws PatternSyntaxException {
```

Comment:
We may want to remove this checked exception. Doesn't need to be changed here necessarily but something that perhaps we should have a chat about.